test/oban/testing_test.exs

```diff
@@ -43,6 +43,27 @@ defmodule Oban.TestingTest do
 
       assert_enqueued worker: Ping, scheduled_at: {seconds_from_now(69), delta: 10}
     end
+
+    test "prints a helpful error message" do
+      insert_job!(%{dest: "some_node"}, worker: Ping)
+
+      try do
+        assert_enqueued worker: Ping, args: %{dest: "other_node"}
+      rescue
+        error in [ExUnit.AssertionError] ->
```

Comment:
Ah, so _this_ is what you were talking about on Slack. Looks great 👍

Reply:
Haha, yeah it was. Was able to find a decent example from the ex_unit tests